src/attr.rs

```diff
@@ -6,13 +6,97 @@ use std::iter;
 #[derive(Debug, Clone, Eq, PartialEq, Hash)]
 pub struct Attribute {
     pub style: AttrStyle,
-    pub value: MetaItem,
+    pub name: Ident,
```

Comment:
Yes, I was initially holding off on it to not make a breaking change. But since `meta_item()` is already going to be one it makes sense to combine them.